### PR TITLE
Bump cdap dependency to 4.3.0

### PR DESCRIPTION
**Pull request by anwar6953**
Also from -SNAPSHOT from the project version.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -80,9 +80,8 @@
   </repositories>
 
   <properties>
-    <cdap.version>4.1.1</cdap.version>
+    <cdap.version>4.3.0.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>4.1.1</cdap.version>
```
Comment (anwar6953) [inline pom.xml]: this was duplicate...

File: pom.xml [hunk 2]
```diff
@@ -80,9 +80,8 @@
   </repositories>
 
   <properties>
-    <cdap.version>4.1.1</cdap.version>
+    <cdap.version>4.3.0.version>
```
Comment (sagarkapare) [inline pom.xml]: nit - `<cdap.version>4.3.0</cdap.version>`

**Comment by sagarkapare** [comment]
lgtm fix and merge

**Comment by sagarkapare** [comment]
LGTM

**Pull request merged**